generators/server/prompts.js

```diff
@@ -53,96 +45,6 @@ function askForServerSideOpts(meta) {
-    //         {
-    //             value: false,
-    //             name: 'No service discovery'
-    //         }
-    //     ],
-    //     default: 'eureka'
-    // },
-    // {
-    //     when: applicationType === 'monolith',
-    //     type: 'list',
-    //     name: 'serviceDiscoveryType',
-    //     message: 'Do you want to use the JHipster Registry to configure, monitor and scale your application?',
-    //     choices: [
-    //         {
-    //             value: false,
-    //             name: 'No'
-    //         },
-    //         {
-    //             value: 'eureka',
-    //             name: 'Yes'
-    //         }
-    //     ],
-    //     default: false
-    // },
-    // {
-    //     when: response =>
-    //         (applicationType === 'monolith' && response.serviceDiscoveryType !== 'eureka') ||
-    //         ['gateway', 'microservice'].includes(applicationType),
-    //     type: 'list',
-    //     name: 'authenticationType',
-    //     message: `Which ${chalk.yellow('*type*')} of authentication would you like to use?`,
-    //     choices: response => {
-    //         const opts = [
-    //             {
-    //                 value: 'jwt',
-    //                 name: 'JWT authentication (stateless, with a token)'
-    //             }
-    //         ];
-    //         if (applicationType === 'monolith' && response.serviceDiscoveryType !== 'eureka') {
-    //             opts.push({
-    //                 value: 'session',
-    //                 name: 'HTTP Session Authentication (stateful, default Spring Security mechanism)'
-    //             });
-    //         }
-    //         if (!reactive) {
-    //             opts.push({
-    //                 value: 'oauth2',
-    //                 name: 'OAuth 2.0 / OIDC Authentication (stateful, works with Keycloak and Okta)'
-    //             });
-    //             if (['gateway', 'microservice'].includes(applicationType)) {
-    //                 opts.push({
-    //                     value: 'uaa',
-    //                     name: 'Authentication with JHipster UAA server (the server must be generated separately)'
-    //                 });
-    //             }
-    //         }
-    //         return opts;
-    //     },
-    //     default: 0
-    // },
-    // {
-    //     when: response =>
-    //         (applicationType === 'gateway' || applicationType === 'microservice') && response.authenticationType === 'uaa',
-    //     type: 'input',
-    //     name: 'uaaBaseName',
-    //     message: 'What is the folder path of your UAA application?',
-    //     default: '../uaa',
-    //     validate: input => {
-    //         const uaaAppData = this.getUaaAppName(input);
-
-    //         if (uaaAppData && uaaAppData.baseName && uaaAppData.applicationType === 'uaa') {
-    //             return true;
-    //         }
-    //         return `Could not find a valid JHipster UAA server in path "${input}"`;
-    //     }
-    // },
```

Comment:
Most of this commented code is related to the JHipster microservice implementation so I don't think we need it since we rely on k8s ;)

And the UAA option is removed in JHipster 7.